example/admin/admin.go

```diff
@@ -3,9 +3,10 @@ package main
 import (
 	"context"
 	"flag"
-	"github.com/zitadel/zitadel-go/v3/pkg/client/middleware"
 	"log"
 
+	"github.com/zitadel/zitadel-go/v3/pkg/client/middleware"
```

Comment:
we normally format the imports as follows:

```
standard lib (e.g. context)

external projects (e.g. github.com/zitadel/oidc/...)

project packages (e.g. github.com/zitadel/zitadel-go/...)
```

see [zitadel gci config](https://github.com/zitadel/zitadel/blob/main/.golangci.yaml#L126-L131)